tests/excmds.test.ts

```diff
@@ -199,8 +198,8 @@ describe("webdriver", () => {
 
             // Then, make sure rsscmd is executed and has the right arguments
             await sendKeys(driver, "<Tab><CR>")
-            await driver.switchTo().parentFrame()
-            const elem = await driver.wait(Until.elementLocated(By.id("rsscmdExecuted")))
+            await (driver.switchTo() as any).parentFrame()
```

Comment:
They removed `parentFrame` from typings, for whatever reason. This still works though